### ISSUE TITLE
[a11y] 9.2.5.3 - Visible labels are part of the accessible name

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5394]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
- **Note:** it is necessary that the visible text is available in the same manner in the accessible name of the control element.
- [x] **All pages:** the visible label "Albert Einstein" is not available in the accessible name of the button (Menuebutton) as it is overwritten by the use of aria-label="Benutzermenü“.

## Status: Rather not fulfilled
- **Page 1 (Alle Dateien)**
- [x] **The visible label "Neu"** is not available in the accessible name of the button as it is overwritten by the use of aria-label.
- [x] **The visible label "öffentlicher Link"** is not available in the accessible name of the button as it is overwritten by the use of aria-label.
- [x] **"Personen hinzufügen"** ("öffentlicher Link") / Dropdown Element "Rolle": the visual label "Rolle" is overwritten by the attribute aria-label="Nach Option suchen" so it is not part of the accessible name.
See general notes

## Status: partly fulfilled
- **Page 2 (Mit anderen geteilten Dateien)**
See general notes
- **Page 3 (Gelöschte Dateien)**
See general notes

- **Page 4 (Konto)**
See general notes

**Comment by pascalwengerter** [comment]
Partly fixed in 4a1c38d587d5ead5aafe439435d3c7c68c8d961f

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10736
